compiler/semantic/op.go

```diff
@@ -678,46 +678,27 @@ func (a *analyzer) semOp(o ast.Op, seq dag.Seq) dag.Seq {
 			Reverse: o.Reverse && len(sortExprs) == 0,
 		})
 	case *ast.Head:
-		val := super.NewInt64(1)
+		val := 1
```

Comment:
Nit: Change this to something other than val since it's is no longer a super.Value.

```suggestion
		count := 1
```